akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
`System.currentTimeMillis` doesn't sound very unique. Have you considered using a sequence number generated by the database, such as `serial` in Postgress or `sequence.nextval` in Oracle? Having a unique position in the event stream is extremely useful for resuming queries (and achieving exactly-once delivery).